gradle/versions.gradle

```diff
@@ -31,6 +31,8 @@ dependencyManagement {
     dependency 'com.google.errorprone:error_prone_core:2.4.0'
     dependency 'com.google.errorprone:error_prone_test_helpers:2.4.0'
 
+    dependency 'com.google.code.findbugs:jsr305:3.0.2'
```

Comment:
This can be dropped, if you also drop the dependency in `ethereum/referencetests/build.gradle`

Reply:
alright, done.